src/django/api/views/facility/facilities_view_set.py

```diff
@@ -1110,6 +1112,8 @@ def merge(self, request):
         merge.delete()
 
         target.refresh_from_db()
+        target.updated_at = timezone.now()
```

Comment:
It's better to use one method like `def update_facility_updated_at_field(facility_id)` in all places to have source of truth.

Reply:
replaced